meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* eP: Thanks for bringing it up. I'll try to attend the all-day hybrid Fed-id CG meeting on Friday. I've seen your demo. Do you think it's ready to deploy it and ask other apps to also adopt it, so we have some implementation? Some signalling/endorsement from LWS WG would also be nice. Jesse also wanted to integrate with the Fediverse, maybe that could be another possibility. And I still need to understand the 'lightweight' FedCM. My understanding was that it also has IDP Registration functionality? I'll have a call with NLNet on Friday. I could help implement the lightweight one, maybe.
+* Th: IdP Registration could benefit from the ecosystem. In univerisities there are a lot of systems that can benefit from the feature. If you want to coordinate and make people aware of that.
+* Th: FedCM can be implemented in production. You need to change cookie policies which would make solidcommunity insecure. 
+* MdJ: Did you talk about that with Aaron Parecki and Sam Goto?
+* Th: Someone found a workaround with two cookie. This would require a lot of work in CSS.
+* MdJ: Are they aware of your work? When I raise the issue they just point to that workaround with a second cookie. It is only for IdP Registration which is not part of the core. https://github.com/w3c-fedid/FedCM/issues/587
+
+### Catalog Awesome List
+
+* JZ: Brief update on discussion Alain and I had about his Awesome List suggestion.  See https://github.com/solid/catalog/issues/6#issuecomment-2787659110
+* JZ: This goes to my point about coordination between CG and Practitioners. As an example of link shaerd in next topic. List of those extensions is useful to people who start servers or who are creating extensions. Catalogue listing can list which servers use which extensions — this is useful primarily to users deciding on which server to select. Another approach is if servers have discovery for which extensions they are supporting - this is useful for developers to know how the app should behave on each server. We need social context not only listing of technical features. If we want to make it useful to the whole community we may need a wider view. Shape for server extension needs feedback (see link above). I hope it is done in a way that it takes into account other types of users. 
+
+### Curated list of known CSS extensions
+
+https://github.com/CommunitySolidServer/CommunitySolidServer/discussions/2024
+
+* eP: Follow up on my action from last week
+
+### Delete backup data from solidcommunity.net before migration to PIVOT
+
+* MdJ: Is that just an announcement of we should move it to next week.
+* AB: The question was made by Jesse. Before deleting it I want to make sure everyone is in agreement.
+* JW: Unless objections I would like those volumes to be deleted.
+* MdJ: Normally we have volumes with backup. Can we keep them for a couple of months and delete them let's say 1st of July. In case some bugs are still found
```

Comment:
```suggestion
* MdJ: Is that just an announcement or should we move it to next week?
* AB: The question was made by Jesse. Before deleting it, I want to make sure everyone is in agreement.
* JW: Barring objections, I would like those volumes to be deleted.
* MdJ: Normally, we have volumes with backup. Can we keep them for a couple of months and delete them, let's say, July 1? In case some bugs are still found.
```